gossip/src/crds_value.rs

```diff
@@ -204,6 +209,47 @@ impl SnapshotHash {
         }
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, AbiExample)]
+pub struct IncrementalSnapshotHash {
+    pub from: Pubkey,
+    pub base_slot: Slot,
+    pub hashes: Vec<(Slot, Hash)>,
+    pub wallclock: u64,
+}
+
+impl Sanitize for IncrementalSnapshotHash {
+    fn sanitize(&self) -> Result<(), SanitizeError> {
+        sanitize_wallclock(self.wallclock)?;
+        if self.base_slot >= MAX_SLOT {
+            return Err(SanitizeError::ValueOutOfBounds);
+        }
+        if self.hashes.len() > MAX_SNAPSHOT_HASHES {
+            return Err(SanitizeError::ValueOutOfBounds);
+        }
```

Comment:
If you want to add such a check here, you need to find the right max value for this new type;
and some tests that the max value is correct. similar to:
https://github.com/solana-labs/solana/blob/d6fdc4b63/gossip/src/cluster_info.rs#L3170-L3194